docs/es/week03/03-3.md

```diff
@@ -0,0 +1,238 @@
+---
+lang: es
+lang-ref: ch.03-3
+title: Propiedades de las señales naturales
+authors: Ashwin Bhola, Nyutian Long, Linfeng Zhang, and Poornima Haridas
+date: 15 February 2020
+translation-date: 02 Apr 2020
+translator: LecJackS
+---
+
+
+<!-- ## [Properties of natural signals](https://www.youtube.com/watch?v=kwPWpVverkw&t=26s)
+-->
+## [Propiedades de las señales naturales](https://www.youtube.com/watch?v=kwPWpVverkw&t=26s)
+
+<!-- All signals can be thought of as vectors. As an example, an audio signal is a 1D signal $\boldsymbol{x} = [x_1, x_2,...,x_T]$ where each value $x_t$ represents the amplitude of the waveform at time $t$. To make sense of what someone is speaking, your cochlea first converts the air pressure vibrations to signals and then your brain uses a language model to convert this signal to a language i.e. it needs to pick the most probable utterance given the signal. For music, the signal is stereophonic which has 2 or more channels to give you an illusion that the sound is coming from multiple directions. Even though it has 2 channels, it's still a 1D signal because time is the only variable along which the signal is changing.
+-->
+Todas las señales pueden considerarse como vectores. Por ejemplo, una señal de audio es una señal 1D \boldsymbol{x} = [x_1, x_2, \cdots, x_T]$ donde cada valor $x_t$ representa la amplitud de la forma de onda en el tiempo $t$. Para dar sentido a lo que alguien está hablando, nuestra cóclea primero convierte las vibraciones de la presión del aire en señales, y luego el cerebro usa un modelo de lenguaje para convertir esta señal a un idioma, es decir, necesita elegir el enunciado más probable, dada la señal. En la música, la señal es estereofónica, o sea que tiene 2 o más canales para dar la ilusión de que el sonido proviene de múltiples direcciones. Aunque tenga 2 canales, sigue siendo una señal 1D porque el tiempo es la única variable a lo largo de la cual está cambiando la señal.
```

Comment:
> señal 1D \boldsymbol{x} 

Aaaaaah! You forgot a `$`!!!!!!!!!!!!!!!!!!!

Reply:
Sorry about this one D: !